lib/ansible/modules/cloud/amazon/rds_v2.py

```diff
@@ -0,0 +1,1148 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+module: rds_v2
+short_description: Manages RDS Instances in AWS
+description:
+  - Creates RDS instances
+  - Modifies RDS instances
+  - Snapshots RDS instances
+  - Restores RDS instances
+  - Deletes RDS snapshots
+  - Deletes RDS instances
+  - Reboots RDS instances
+version_added: "2.3"
+requirements: [ boto3 ]
+options:
+  command:
+    description:
+        - specifies the action to take
+        - absent to remove resource
+    required: true
+    default: None
+    choices: [ 'create', 'delete', 'modify', 'snapshot', 'reboot', 'restore' ]
+  db_instance_identifier:
+    description:
+      - Database instance identifier. Required except when using command=delete on just a snapshot.
+    required: false
+    default: null
+  engine:
+    description:
+      - The name of the database engine to be used for this instance.  Used only when command=create or restore.
+    required: false
+    default: null
+    choices: [ 'mariadb', 'MySQL', 'oracle-se1', 'oracle-se', 'oracle-ee', 'sqlserver-ee', 'sqlserver-se', 'sqlserver-ex', 'sqlserver-web', 'postgres', 'aurora']
```

Comment:
Is there a way to find this out by introspecting in boto3 instead? Manually updating this is kind of a pain.